src/widget/wcoverartmenu.cpp

```diff
@@ -82,6 +83,64 @@ void WCoverArtMenu::slotChange() {
     coverInfo.source = CoverInfo::USER_SELECTED;
     coverInfo.coverLocation = selectedCoverPath;
     coverInfo.setImage(image);
+
+    if (m_pConfig->getValue<bool>(mixxx::library::prefs::kCreateCopyOfTheCoverArtConfigKey)) {
+        QString savedCoverArtLocation = fileInfo.absoluteDir().path() + "/" +
+                fileInfo.completeBaseName() + ".jpg";
+        if (QFile::exists(savedCoverArtLocation)) {
+            QMessageBox saveBox(
+                    QMessageBox::Question,
+                    tr("Overwrite Existing Cover Art?"),
+                    tr("The track already has a related Cover Art in same the path\n"
```

Comment:
"The file already exists ... " the cover art guesser looks for more locations, I consider all as related.